app/uk/gov/hmrc/gform/auth/models/AuthResult.scala

```diff
@@ -22,6 +22,7 @@ sealed trait AuthResult
 final case class AuthSuccessful(retrievals: MaterialisedRetrievals) extends AuthResult
 final case class AuthRedirect(loginUrl: String, flashing: Seq[(String, String)] = Seq.empty) extends AuthResult
 final case class AuthAnonymousSession(redirectUrl: Call) extends AuthResult
+final case class AuthAWSALBSession(redirectUrl: Call) extends AuthResult
```

Comment:
This is never used, so apparently it is not needed, so should be removed?

Reply:
Yeah should be removed.